that is really strange ... this define creates conflicting types between
(f)truncate and (f)truncate64 can be seen in curl ...

--- ./include/unistd.h~	Sat Apr 19 17:25:19 2003
+++ ./include/unistd.h	Sat Apr 19 20:06:52 2003
@@ -123,8 +123,8 @@
 #if defined _FILE_OFFSET_BITS && _FILE_OFFSET_BITS == 64
 #define open open64
 #define creat creat64
-#define truncate truncate64
-#define ftruncate ftruncate64
+/*#define truncate truncate64
+#define ftruncate ftruncate64*/
 #define getdents getdents64
 #endif
 
